src/c++/uds/kernelLinux/tests/getTestIndexNames.c

```diff
@@ -10,16 +10,25 @@
 
 #undef VDO_USE_ALTERNATE
 #undef VDO_USE_ALTERNATE_2
-#ifdef RHEL_RELEASE_CODE
+#undef VDO_USE_ALTERNATE_3
+
+/*
+  COMMENT HERE
+ */
+#if defined(RHEL_RELEASE_CODE)
 #define VDO_USE_ALTERNATE
```

Comment:
What about the code to have the fake RHEL_RELEASE_CODE 9.99? Did it turn out that you don't need it?